### ISSUE TITLE
Switch from novoda-bintray-plugin

### ISSUE THREAD

**Issue by lisawray** [lisawray/groupie#216]
I was using novoda-bintray-plugin to publish to Bintray and from there to maven central, but Novoda's plugin is currently broken: https://github.com/novoda/bintray-release/issues/177 

Took a shot at switching it over to https://github.com/dcendents/android-maven-gradle-plugin (working example: https://github.com/trello/RxLifecycle) on branch https://github.com/lisawray/groupie/tree/publishing ....

but I haven't got it working yet.  Would love some eyes from anyone familiar with publishing directly to maven central.

```
./gradlew clean build uploadArchives

> Task :library:uploadArchives FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':library:uploadArchives'.
> Could not publish configuration 'archives'
   > Failed to deploy artifacts: Could not transfer artifact com.xwray:groupie:aar:2.2.0 from/to remote (https://oss.sonatype.org/service/local/staging/deploy/maven2/): Failed to transfer file: https://oss.sonatype.org/service/local/staging/deploy/maven2/com/xwray/groupie/2.2.0/groupie-2.2.0.aar. Return code is: 401, ReasonPhrase: Unauthorized.
```

**Comment by bernaferrari** [comment]
It was really simple when I did it:
https://github.com/bernaferrari/EmojiSlider/blob/master/emojislider/build.gradle

https://medium.com/@yegor_zatsepin/simple-way-to-publish-your-android-library-to-jcenter-d1e145bacf13
https://android.jlelse.eu/publishing-your-android-kotlin-or-java-library-to-jcenter-from-android-studio-1b24977fe450

**Comment by lisawray** [comment]
That's the article I was reading this morning :) I think I've almost got
it, I'll push tomorrow

**Comment by jordond** [comment]
Another simple option is [jitpack](https://jitpack.io/#lisawray/groupie).

**Comment by bernaferrari** [comment]
But jitpack requires updating the build.gradle repositories, while jcenter
"just works".

**Comment by lisawray** [comment]
Anything would be fine at this point.  I'm at my wit's end.  I can't even get it to work with jitpack, and I think it's because it's multiple libraries.

It's here if anyone wants to give it a shot: https://github.com/lisawray/groupie/tree/jitpack

**Comment by bernaferrari** [comment]
Did you try this?

https://medium.com/mindorks/distribute-multiple-module-library-on-bintray-for-java-and-android-developers-212216bc1aa7

FastAdapter follows this pattern and even includes a script:
https://github.com/mikepenz/FastAdapter/blob/develop/release.sh

**Comment by lisawray** [comment]
Done.  Published as 2.2

It hasn't synced to jcenter yet, so for the next few hours you'll have to use 

```
    repositories {
        maven { url 'https://dl.bintray.com/lisawray/maven'}
    }
```

**Comment by mr-archano** [comment]
@lisawray we have just released a new version of [`novoda/bintray-release`](https://bintray.com/novoda/maven/bintray-release/0.9) that should fix the issue you were facing.
It was a tricky one to fix, it took longer than anticipated. Feel free to give it a go and open an issue if any problem.
